### DISCUSSION TITLE
[SOLVED] - Are these missing sounds?

### DISCUSSION THREAD

**Discussion by virgo77** [azerothcore/azerothcore-wotlk#13534] [General]
Hi all!
Because I don't know if it's a core issue or not, i do prefer to open a discussion before an issue ticket.
I was wondering about some sounds like the 'ding ding' when the Alliance's boats reached the dock or the long 'hornet' sound of the Horde's Flights when docking too... and when my cat pet meows when it is bored and stretches its legs.
For me, these sounds are "missing". They don't play. I can't hear them. I tried lots of different clients, but still the issue. I tried to install a fresh WotLK Client from DVD and applying all the patches and, still nothing. And I'm sure that these sounds are present.

Some advices?

Thanks
V.

**Comment by virgo77** [comment]
Hi, i 'bump' this topic because I didn't had replies yet.
Please, can someone tell me if these sounds play on his computer?
Thanks a lot :)
V.

**Comment by heyitsbench** [comment]
The boat docking noises are currently not sent by the server. This is a known issue [here](https://github.com/azerothcore/azerothcore-wotlk/issues/11340) and not an issue with your client. I don't personally know myself about the horde zeppelins or the pet idle noises, since I don't recall them myself, but opening an issue about them might bring more attention to it.

**Comment by virgo77** [comment]
Hi HeyItsBench!
Thanks for your reply. I wasn't aware that it was an old issue. As there are lots of issues, I didn't search for this one a lot. And as I'm not always good in english, I didn't knew which keywords to use. So thanks a lot :)
It's good to know that the problem do not come from the client.
I think I will do as you say. I will create a new issue.
Thanks again.
V.
